service/controller/v1/resource/configmap/delete.go

```diff
@@ -3,16 +3,30 @@ package configmap
 import (
 	"context"
 
-	"github.com/giantswarm/operatorkit/resource/crud"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/controller"
 )
 
-// NewDeletePatch is a no-op.
-// We do not want to delete the configmap, as the running prometheus relies on it.
-func (r *Resource) NewDeletePatch(ctx context.Context, obj, currentState, desiredState interface{}) (*crud.Patch, error) {
-	return nil, nil
+// NewDeletePatch calls NewUpdatePatch as the ConfigMap must be updated with
+// removed cluster. This is important when the last cluster in the
+// installation is removed.
+func (r *Resource) NewDeletePatch(ctx context.Context, obj, currentState, desiredState interface{}) (*controller.Patch, error) {
```

Comment:
I think `controller.Patch` has to be `crud.Patch` to align with other operators.